src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/DecryptTests.cs

```diff
@@ -21,7 +21,9 @@ namespace System.Security.Cryptography.Pkcs.EnvelopedCmsTests.Tests
 {
     public static partial class DecryptTests
     {
-        [Fact]
+        public static bool IsNet462OrNewer() => (!PlatformDetection.IsFullFramework || PlatformDetection.IsNetfx462OrNewer());
```

Comment:
@krwq No, IsNetfx462OrNewer shouldn't return true for CoreFX.  Because it isn't netfx.

But that's also why these should be describing the failing functionality, not just a version number.